modules/vmss/variables.tf

```diff
@@ -44,6 +44,11 @@ variable "create_public_pip" {
   type    = bool
 }
 
+variable "public_ip_prefix_id" {
```

Comment:
Please run `pre-commit run -a` on the repository. I see that the README.md file did not get updated with the new variable of yours.